.github/workflows/release.yml

```diff
@@ -3,6 +3,7 @@ name: release
 on:
   push:
     tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
What's the reason for this? I'm assuming that a tool we're using creates versions without the `v` prefix?

Reply:
yeah apparently the newer versions of `cargo release` don't use the `v` prefix :(

Reply:
actually i'm going to revert this because the new installers assume a `v` prefix